src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -372,4 +379,29 @@ mod tests {
         assert!(ku.encipher_only());
         assert!(ku.decipher_only());
     }
+
+    #[test]
```

Comment:
We don't usually write these like this, because they're not _really_ testing anything. They just ensure that the structure we wrote works with rust-asn1.

My suggestion would be to remove the `#[derive(asn1::Asn1Read, asn1::Asn1Write)]` from the type, and remove this test. And then you can add the derive back in a follow up PR.